### PR TITLE
[s] hotfix

### PR DESCRIPTION
**Pull request by monster860**
Hotfix for Href Exploit

#### Changelog

:cl:  
bugfix: Hotfix for Href Exploit
/:cl:

### CODE REVIEW

**Review by JamieD1 — APPROVED** [review]

**Comment by KingTerrytheTerrible** [comment]
lmfao

**Comment by KingTerrytheTerrible** [comment]
fucking regexes ahahahahaha

**Comment by honkhonkclown** [comment]
GNAA PRIDE STATIONWIDE

File: code/modules/goonchat/browserOutput.dm [hunk 1]
```diff
@@ -203,6 +203,11 @@ GLOBAL_DATUM_INIT(iconCache, /savefile, new("tmp/iconCache.sav")) //Cache of ico
 		return
 
 	if(cookie != "none")
+		var/regex/crashy_thingy = /new regex("(\\\[ *){5}")
+		if(crashy_thingy.Find(cookie))
+			message_admins("[key_name(src.owner)] tried to crash the server using at least 5 "\[" in a row")
+			log_admin_private("[key_name(owner)] tried to crash the server using at least 5 "\[" in a row")
```
Comment (Exxion) [inline code/modules/goonchat/browserOutput.dm:209]: dude unescaped quotation marks lmao

**Pull request merged**